angular.json

```diff
@@ -29,10 +29,12 @@
             "styles": [
               "./node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css",
               "node_modules/normalize.css/normalize.css",
+              "node_modules/materialize-css/dist/css/materialize.css",
               "src/styles/base.scss",
               "./node_modules/froala-editor/css/froala_editor.pkgd.min.css",
               "./node_modules/froala-editor/css/froala_style.min.css",
-              "./node_modules/ng-pick-datetime/assets/style/picker.min.css"
+              "./node_modules/ng-pick-datetime/assets/style/picker.min.css",
+              "src/styles/styles-port/basic.scss"
```

Comment:
Didn't we agree on removing `styles-port`?

Reply:
I am making a PR for that after merging that It will be removed. I kept it to make the PR functional.